modules/ob-multi-platform-builds.adoc

```diff
@@ -0,0 +1,9 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="multi-platform-builds_{context}"]
+= Multi-platform builds
+
+The sample build strategy contains a `platforms` array parameter that you can set to use the `buildkit` strategy for building multi-platform images. If you do not set this array parameter, the image is built for the platform that is supported by the `FROM` image defined in your build strategy resource. If that image supports multiple platforms, then the image will be built for the platform of your cluster node.
```

Comment:
Avoid future tense: " ... then the image is built for the platform of your cluster node."